.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -6,10 +6,6 @@ name: Deploy to Firebase Hosting on PR
 jobs:
   build_and_preview:
     if: '${{ github.event.pull_request.head.repo.full_name == github.repository }}'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - run: npm ci && npm run build
```

Comment:
You changed more than one line. Just remove `- run: npm ci && npm run build` this line and put everything else back.